doc/lua_api.md

```diff
@@ -8781,6 +8781,8 @@ child will follow movement and rotation of that bone.
         * `type`: Available types:
             * `"regular"`: Uses 0 textures, `base_color` ignored
             * `"skybox"`: Uses 6 textures, `base_color` used as fog.
+            * `"skybox_back"`: Uses 6 textures, `sky_color` used as fog, stars / sun / moon in foreground.
```

Comment:
Might also be worth documenting that `skybox_back` / `skybox_front` will require a Luanti client version of 5.13+